question/bank/previewquestion/preview.php

```diff
@@ -266,25 +276,10 @@
 }
 $previewdata['techinfo'] .= print_collapsible_region_end(true);
 
-// Output a link to export this single question.
```

Comment:
why is this removed? its a feature

Reply:
It wasn't part of the wireframe, I put it back.